### ISSUE TITLE
Line has no descriptive tags

### ISSUE THREAD

**Issue by SomeoneElseOSM** [openstreetmap/iD#10455]
### URL

https://www.openstreetmap.org/edit?editor=id#map=19/53.986065/-1.049651

### How to reproduce the issue?

In the real world, something has been removed.  It still exists in imagery.  Naturally, I want to remove it from OSM, but I also want to leave something on that geometry so that armchair mappers don't "just add it back".  An attempt to do this via a note tag results in an "error" (see below).  In order to complete the edit I need to add some other nonsensical tags and remove them in another editor.

What should happen instead is that a "note" tag on its own should be valid, rather than having to add some garbage and remove it with another editor.  See e.g. https://www.openstreetmap.org/way/1313248451/history

### Screenshot(s) or anything else?

<img width="180" alt="Screenshot 2024-09-04 143618" src="https://github.com/user-attachments/assets/d7b4ca78-e194-4ad2-8d4e-9551862b17ed">

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.30.2

### Which browsers are you seeing this problem on?

Firefox

**Comment by k-yle** [comment]
I think the idea behind this warning is to encourage you to use `was:highway=footway` or some other lifecycle prefix, right? 

The message is just not very helpful....

**Comment by maro-21** [comment]
Exactly. Every element should have tags, even with `disused:`, `abandoned:` or `was:` prefixes.

**Comment by SomeoneElseOSM** [comment]
Really? What would be appropriate for https://www.openstreetmap.org/way/637258230#map=15/54.35869/-0.94564 ?

**Comment by maro-21** [comment]
`not:highway=path` :]
[https://wiki.openstreetmap.org/wiki/Key:not:*](https://wiki.openstreetmap.org/wiki/Key:not:*)

**Comment by tyrasd** [comment]
I would agree that the use of lifecycle tags (e.g. `abandoned:*`, `razed:*`, `was:*`, `not:*` etc.) would be preferred for these.
